tests/utils_tests/models.py

```diff
@@ -11,3 +11,11 @@ def next(self):
 class Thing(models.Model):
     name = models.CharField(max_length=100)
     category = models.ForeignKey(Category, models.CASCADE)
+
+
+class FooThing(models.Model):
```

Comment:
any reason not to reuse `Category` instead of this new model? (Each model added makes the test suite a bit slower, so we try to reuse where possible -- not sure if the custom `next()` method that it has is an issue or not).